cmd/server/http/http.go

```diff
@@ -50,8 +50,9 @@ func NewServer(opts *Options, slackClient *slack.Client, flowSvc *flow.Service,
 	mux.HandleFunc("/policies/", trace(tracer, authenticate(opts.HamCtlAuthToken, policy(&payloader, policySvc))))
 	mux.HandleFunc("/describe/", trace(tracer, authenticate(opts.HamCtlAuthToken, describe(&payloader, flowSvc))))
 	mux.HandleFunc("/webhook/github", trace(tracer, githubWebhook(&payloader, flowSvc, policySvc, gitSvc, slackClient, opts.GithubWebhookSecret)))
-	mux.HandleFunc("/webhook/daemon", trace(tracer, authenticate(opts.DaemonAuthToken, daemonWebhook(&payloader, flowSvc))))
```

Comment:
This is a breaking change in case you are unaware. Daemons will fail during rollout until both the release manager and daemons are up on the new version.

I don't think its a problem here as we control both apps, but we cannot keep the old daemon running in some of the environments.

We could keep the old endpoint along with the handler maybe? To give us selves some more air during the deployment for testing it in only dev?

Reply:
Do you think it's worth the work to support this? @Crevil